benchmarks/benchmark/tools/profile-generator/container/benchmark_serving.py

```diff
@@ -321,21 +406,40 @@ async def benchmark(
             args.sax_model,
             model,
         )
-    )
+      )
+    else: 
+      task = asyncio.create_task(
+      send_request(
+          args.backend,
+          api_url,
+          prompt,
+          prompt_len,
+          output_len,
+          args.best_of,
+          args.use_beam_search,
+          args.top_k,
+          tokenizer,
+          args.sax_model,
+          model,
+        )
+      )
```

Comment:
Something like functools.partial may make this cleaner, but not blocking

Reply:
Mmm functools.partial would require only using one function for both streaming and non-streaming requests, and hardcoding one of the values with partial. I think having send_request and send_request_streaming be the same function with different implementations allows for easier refactoring in the future. What do you think?